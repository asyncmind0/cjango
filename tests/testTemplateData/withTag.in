A variable: {{ foo }}
A second variable: {{ bar }}
{% with foo as baz %}
An alias to the first variable: {{ baz }}
    {% with bar as foo %}
An alias to the second variable: {{ foo }}
    {% endwith %}
{% endwith %}
The first alias out ot scope: {{ baz }}
The first variable again: {{ foo }}
