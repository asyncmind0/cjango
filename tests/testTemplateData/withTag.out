A variable: 1
A second variable: 2

An alias to the first variable: 1
    
An alias to the second variable: 2
    

The first alias out ot scope: 
The first variable again: 1
